src/stactools/noaa_sst/stac.py

```diff
@@ -30,77 +41,78 @@ def create_collection() -> Collection:
+        description=DESCRIPTION,
+        license=LICENSE,
+        providers=[SST_PROVIDER],
         extent=extent,
         catalog_type=CatalogType.RELATIVE_PUBLISHED,
     )
 
+    collection.add_link(LICENSE_LINK)
+
     return collection
 
 
-def create_item(asset_href: str) -> Item:
+def create_item(nc_href: str, sst_cog_href: str, sif_cog_href: str) -> Item:
     """Create a STAC Item
-
-    This function should include logic to extract all relevant metadata from an
-    asset, metadata asset, and/or a constants.py file.
-
-    See `Item<https://pystac.readthedocs.io/en/latest/api.html#item>`_.
-
+    Collect metadata from a NOAA-SST netcdf file to create the Item
     Args:
-        asset_href (str): The HREF pointing to an asset associated with the item
-
+        nc_href (str): The HREF pointing to the NOAA netcdf file
+        cog_href (str): The HREF pointing to the associated asset COG. The COG should
+        be created in advance using `cog.create_cog`
     Returns:
         Item: STAC Item object
     """
-
-    properties = {
-        "title": "A dummy STAC Item",
-        "description": "Used for demonstration purposes",
-    }
-
-    demo_geom = {
+    with Dataset(nc_href) as ds:
+        properties = {
+            "title": ds.title,
+            "noaa-sst:institution": ds.institution,
+            "noaa-sst:source": ds.source,
+            "noaa-sst:history": ds.history,
+            "noaa-sst:comment": ds.comment,
+        }
+        item_datetime = datetime.strptime(ds.time_coverage_start,
+                                          '%Y%m%dT%H%M%SZ')
+
+        dims = ds.dimensions
+        ds_shape = [dims["lon"].size, dims["lat"].size]
+        x_cellsize = 360.0 / float(dims["lon"].size)
+        y_cellsize = 180.0 / float(dims["lat"].size)
+
+    global_geom = {
         "type":
         "Polygon",
-        "coordinates": [[[-180, -90], [180, -90], [180, 90], [-180, 90],
-                         [-180, -90]]],
+        "coordinates": [[[-180.0, -90.0], [180.0, -90.0], [180.0, 90.0],
+                         [-180.0, 90.0], [-180.0, -90.0]]],
     }
 
-    # Time must be in UTC
-    demo_time = datetime.now(tz=timezone.utc)
-
-    item = Item(
-        id="my-item-id",
-        properties=properties,
-        geometry=demo_geom,
-        bbox=[-180, 90, 180, -90],
-        datetime=demo_time,
-        stac_extensions=[],
-    )
+    item = Item(id=f"{NOAA_SST_ID}-noaa-{item_datetime}",
+                properties=properties,
+                geometry=global_geom,
+                bbox=SPATIAL_EXTENT,
+                datetime=item_datetime,
+                stac_extensions=[])
 
+    proj_attrs = ProjectionExtension.ext(item, add_if_missing=True)
+    proj_attrs.epsg = SST_EPSG
+    proj_attrs.bbox = SPATIAL_EXTENT
+    proj_attrs.shape = ds_shape
+    proj_attrs.transform = [
+        SPATIAL_EXTENT[0],
+        x_cellsize,
+        0.0,
+        SPATIAL_EXTENT[1],
+        0.0,
+        -y_cellsize,
+    ]
```

Comment:
This sets the proj extension but is immediately overwritten below

Reply:
Removed the latter block